SonarQube.Client.Tests/SonarQubeService_GetSuppressedIssuesAsync.cs

```diff
@@ -45,16 +47,12 @@ public async Task GetSuppressedIssuesAsync_Old_ExampleFromSonarQube()
         {
             await ConnectToSonarQube();
 
-            ////var issues = ServerIssue.Parser.ParseDelimitedFrom(new FileStream(@"TestResources\IssuesProtobufResponse", FileMode.Open));
-
             using (var reader = new StreamReader(@"TestResources\IssuesProtobufResponse"))
             {
                 SetupRequest("batch/issues?key=project1", reader.ReadToEnd());
             }
-            //SetupRequest("batch/issues?key=project1",
-            //    new StreamReader(@"TestResources\IssuesProtobufResponse").ReadToEnd());
 
-            var result = await service.GetSuppressedIssuesAsync("project1", CancellationToken.None);
+            var result = await service.GetSuppressedIssuesAsync("project1", null, CancellationToken.None);
```

Comment:
The `branch` parameter is optional, so I modified all of the existing tests to pass `null`.